_data/es/settings.yml

```diff
@@ -44,6 +44,8 @@ contact_page:
       'Department of Energy': Departamento de Energía
       'Department of the Interior': Departamento del Interior
       'Department of Labor': Departamento del Trabajo
+      'Department of State': Department of State
```

Comment:
these havent been translated yet. is this not ready yet?

Reply:
@mdiarra3 awaiting confirmation of not only translations but all the values themselves